release_notes/47-release-notes.adoc

```diff
@@ -553,4 +553,18 @@ This release addresses the following security vulnerability:
 //ROX-30909
 * Lack of a data size check in the Axios dependency (link:https://access.redhat.com/security/cve/CVE-2025-58754[CVE-2025-58754])
 
+[id="about-release-479_{context}"]
+== About release version 4.7.9
+
+*Release date*: {ga-date-479}
+
+This release addresses the following bug fix:
+
+//ROX-31554
```

Comment:
Shouldn't we be consistent with what was used in the 4.9.1 release notes?

* Fixed an issue that caused Central to panic and terminate Sensor connections when a Sensor sent an event type that was unknown to Central. This issue occurred specifically when Sensor version 4.9 ran with Central version 4.7 or 4.8 on an {ocp-virt} cluster. The fix ensures Central operates normally under these conditions and also improves future compatibility between Central version 4.9 and Sensor.